wp/lib/bap_wp/src/precondition.ml

```diff
@@ -388,17 +340,16 @@ let spec_verifier_error (sub : Sub.t) (_ : Arch.t) : Env.fun_spec option =
   else
     None
 
-let spec_verifier_assume (sub : Sub.t) (arch : Arch.t) : Env.fun_spec option =
+let spec_verifier_assume (sub : Sub.t) (_ : Arch.t) : Env.fun_spec option =
   if Sub.name sub = "__VERIFIER_assume" then
-    let offset = get_stack_ptr_offset sub arch in
     let open Env in
     Some {
       spec_name = "spec_verifier_assume";
       spec = Summary
           (fun env post tid ->
              let ctx = Env.get_context env in
              let post = set_fun_called post env tid in
-             let post, env = increment_stack_ptr post env offset in
+             let post, env = increment_stack_ptr post env in
```

Comment:
This line seems to be repeated in every spec. I wonder if we'll be adding anything to this in the future? If so, we should be creating a `function_epilogue` or something, that handles this kind of book-keeping. Probably premature, but something to keep in mind.